SECURITY.md

```diff
@@ -0,0 +1,32 @@
+# Security Policy
+
+## Supported Versions
+
+The following versions of tsdf are currently supported with security updates:
+
+| Version | Supported          |
+| ------- | ------------------ |
+| 0.4.x   | :white_check_mark: |
+| < 0.4   | :x:                |
+
+## Reporting a Vulnerability
+
+If you discover a security vulnerability in tsdf, please report it by sending an email to p.kok[at]esciencecenter.nl. We ask that you do not publicly disclose the issue until it has been addressed.
+
+In your report, please provide the following information:
+
+- A brief description of the vulnerability.
+- Steps to reproduce the vulnerability.
+- Versions of tsdf affected.
+
+We are committed to responding promptly to security reports. We will acknowledge your email within 48 hours and will strive to provide a timeline for resolution as soon as possible.
```

Comment:
I'm not sure if we can promise this after the end of the project, as it's not certain there will be budget for that. Also, if we promise 48 hours, there should be more people in the team, taking into account holidays and such.